File: PYTHON/pegawai.py
from orang import Orang

class Pegawai(Orang):
    def __init__(self, nik, nama, jenis_kelamin, id_pegawai, departemen, tahun_masuk):
        super().__init__(nik, nama, jenis_kelamin)
        self.id_pegawai = id_pegawai
        self.departemen = departemen
        self.tahun_masuk = tahun_masuk

    def get_id_pegawai(self): return self.id_pegawai
    def get_departemen(self): return self.departemen
    def get_tahun_masuk(self): return self.tahun_masuk

File: JAVA/Pegawai.java
public class Pegawai extends Orang {
    private String idPegawai;
    private String departemen;
    private int tahunMasuk;

    public Pegawai() {}

    public Pegawai(String nik, String nama, String jenisKelamin, String idPegawai, String departemen, int tahunMasuk) {
        super(nik, nama, jenisKelamin); // Memanggil constructor parent (Orang)
        this.idPegawai = idPegawai;
        this.departemen = departemen;
        this.tahunMasuk = tahunMasuk;
    }

    public String getIdPegawai() { return idPegawai; }
    public String getDepartemen() { return departemen; }
    public int getTahunMasuk() { return tahunMasuk; }
}